Makefile

```diff
@@ -259,7 +259,7 @@ test-unit-with-coverage:
 ## Setup the environment for the acceptance tests
 test-acceptance-setup: setup-venv
 ifeq ($(TEST_ACCEPTANCE_START_SBO), local)
-test-acceptance-setup: test-cleanup create-test-namespace deploy-test-3rd-party-crds set-test-namespace deploy-rbac deploy-crds
+test-acceptance-setup: test-cleanup create-test-namespace deploy-test-3rd-party-crds set-test-namespace deploy-rbac deploy-crds build
```

Comment:
let's make `build` first in the list so that we can fail fast.

Reply:
good point